Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -59,7 +59,7 @@ class DatabaseContainer: NSPersistentContainer {
     static var cachedModel: NSManagedObjectModel?
 
     /// All `NSManagedObjectContext`s this container owns.
-    private lazy var allContext: [NSManagedObjectContext] = [viewContext, backgroundReadOnlyContext, writableContext]
+    private(set) lazy var allContext: [NSManagedObjectContext] = [viewContext, backgroundReadOnlyContext, writableContext]
```

Comment:
Adding `s` to the `allContext` property would be nice but not an important issue here.

Reply:
In order to not re-run the CI, I'll leave this one for now 👍